cobigen-angular-client-generation/index.asciidoc

```diff
@@ -0,0 +1,94 @@
+= Generate Angular Client with devonfw
+== angular client generation with cobigen
+====
+With the CobiGen code generator you can easily generate components for your angular application.
+The following tutorial provides an example for using the CobiGen cli.
+You can find more information on https://github.com/devonfw/cobigen/wiki/howto_angular-client-generation
+## Prerequisites
+* devonfw Ide
+* Install yarn globally
+## Learning goals.
+Here in this tutorial we will learn the following:
+* Create an angular project
+* Cobigen to generate the angular source code out of the java files
+* Change the user interface in the angular project to show the new components
+* Show the newly created components
+====
+
+[step]
+--
+restoreDevonfwIde(["java", "mvn", "node", "ng", "npm"])
+--
+
+For this tutorial an installation of the devonfw IDE is required, which was already set up for you. The next step is to install the CobiGen CLI.
+[step]
+--
+installCobiGen()
+--
+
+CobiGen uses the backend classes for generating your code used in the angular application. So in the next steps you will create a java project, create a simple java class and run the backend server.
+[step]
+--
+createDevon4jProject("cobigenjavaexample")
+--
+
+====
```

Comment:
Opening with `====` indicates that you'd like to add a new step with prepended and appended comments. However, regarding the [tutorial development section](https://github.com/devonfw-tutorials/tutorials/wiki/Development) you can see that this block is opened with `====`, then you add the before-step instructions, then the step itself and then the result message afterwards.